sopel/modules/reddit.py

```diff
@@ -13,8 +13,8 @@
 import html
 import re
 
-import praw
-import prawcore
+import praw  # type: ignore
+import prawcore  # type: ignore
```

Comment:
Ditto what I said about bugzilla's ignore comment for `xmltodict`. Only the expected error should be ignored.